### ISSUE TITLE
More target elements for engineApplyShaderToWorldTexture

### ISSUE THREAD

**Issue by Castillos15** [multitheftauto/mtasa-blue#1111]
**Is your feature request related to a problem? Please describe.**
Currently engineApplyShaderToWorldTexture only supports vehicles, objects and peds for per-element shader applying.

**Describe the solution you'd like**
I would like engineApplyShaderToWorldTexture to also support applying shaders in a per-element basis to blips.

**Describe alternatives you've considered**
The only other way to achieve what I want (adding new blips) is by either creating an entire new radar, or by adding custom blip support to default GTA radar, but having the possibility to just replace current blip images using shaders would make it a lot easier.

**Additional context**

It would be nice to see other visual elements available as well:

-Effects ("Very and most important")
-Searchlights
-Markers
-Pickups
-Blips
-Projectiles

**Comment by PlatinMTA** [comment]
You can apply shaders to blips, but i never tried applying shaders to blips by their element, I dont even know if that works.

If that is impossible for the moment, then yes this would be a nice addition.

**Comment by Castillos15** [comment]
> You can apply shaders to blips, but i never tried applying shaders to blips by their element, I dont even know if that works.
> 
> If that is impossible for the moment, then yes this would be a nice addition.

Yes, I know you can apply shaders to blips (I already do it in one of my scripts).
But what you can't do at the moment is apply a shader to specific blip elements (I tried).

**Comment by Einheit-101** [comment]
We could close this issue in favor of my newly created one. There is no reason to keep this open.
https://github.com/multitheftauto/mtasa-blue/issues/1514

**Comment by ImSkully** [comment]
Bumping for visibility, hoping someone can implement this for additional element types *(specifically blips and pickups)*, would be a great addition to 1.6.0.

**Comment by PlatinMTA** [comment]
#3973 i wonder if we only need to add pickups and markers to the pools like in that PR... that looks fairly easy to implement
